business/checkers/virtual_service_checker.go

```diff
@@ -33,7 +35,7 @@ func (in VirtualServiceChecker) Check() models.IstioValidations {
 func (in VirtualServiceChecker) runIndividualChecks() models.IstioValidations {
```

Comment:
I think the VirtualServicesChecker is the best checker to explain a bit the architecture of the validations.
One ObjectChecker runs two different validations: `IndividualChecks` and `GroupChecks`.
- `IndividualChecks` are all the checkers that are focused on one single Object. For example, checking if the `exportTo` field has valid values. It doesn't consider other Istio Objects to see whether there are overlaps between them. Just one object and other resources to determine its validity.
- `GroupChecks` are all the checkers that need a group of objects of the same type to determine its validity. It might end up with one validation for more than one object. For instance, [KIA0201](https://kiali.io/documentation/latest/validations/#_kia0201_more_than_one_destinationrules_for_the_same_host_subset_combination). It need to parse a group of DRs in order to see that there are multiple ones defined for the same host.

Reply:
Therefore, appending both exported VS to the subject of analysis VS will make Kiali validate both sets of VS. And we just want to analyze the VS living in the current ns aka `in.VirtualServices`.